CONTRIBUTING.adoc

```diff
@@ -1,6 +1,9 @@
 :product: App Services
```

Comment:
```suggestion
:product: Application Services
```
Not sure if this one should be spelled out in full?

Reply:
Don't think so. At least it has always been called App Services, deliberately AFAIK.